docs/collecting-data/code-generation/snowtype-config/index.md

```diff
@@ -44,6 +44,38 @@ The target language to generate the required code for. [See list of available la
 
 The outpath relative to the current working directory when running the script.
 
+### `options`
+
+Options of any kind that are related to Snowtype behaviour. The following TypeScript type describes the current available options:
```

Comment:
when I see `any` it make me feel bad 🤣 (just joking) I think the text could read a bit more concise like this

```
Options related to Snowtype behavior are described by the following TypeScript type:
```

Reply:
Added properly! Thanks!